### PR TITLE
Use centos stream 9 as base image

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

File: Dockerfile
Comment (dtantsur) [inline Dockerfile:3]: (Thinking aloud) we should probably remove the whole `RUN`, we don't use any packages in this container.

**Comment by kashifest** [comment]
/hold
Centos tests didnt run properly. CI has a glitch it seems.

**Comment by furkatgofurov7** [comment]
> /hold Centos tests didnt run properly. CI has a glitch it seems.

Retriggered the tests after a fix in project infra and it looks fine now, unholding it.
/hold cancel

**Review by Rozzii — APPROVED** [review]

**Review by furkatgofurov7 — APPROVED** [review]

**Comment by fmuyassarov** [comment]
LGTM. I don't have rights to give lgtm label though.

**Pull request merged**
